joint_trajectory_controller/test/test_trajectory_controller.cpp

```diff
@@ -428,25 +318,28 @@ TEST_P(TrajectoryControllerTestParameterized, correct_initialization_using_param
     EXPECT_LE(0.0, joint_eff_[2]);
   }
 
-  // cleanup
-  state = traj_controller_->get_node()->cleanup();
-  ASSERT_EQ(State::PRIMARY_STATE_UNCONFIGURED, state.id());
+  // reactivated
+  // wait so controller process the third point when reactivated
+  std::this_thread::sleep_for(std::chrono::milliseconds(3000));
+  // TODO(anyone) test copied from ROS 1: it fails now!
+  //  should the old trajectory really be processed after reactivation?
```

Comment:
No, I think after reactivation, we should remove the trajectory that was commanded. It is basically “a reset”

Reply:
Fixed with bceaf96e93dd2b14c3f1ee3ec00847ac4d41f5cb in #705

Reply:
And even more with 0bf984488008d8a4e1bd2b26fd319fc870338cde in #558